src/app/core/project-service/project-service.service.ts

```diff
@@ -1,8 +1,74 @@
 import { Injectable } from '@angular/core';
+import { Headers, Http, Response } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/catch';
+import 'rxjs/add/observable/throw';
+import 'rxjs/add/operator/do';
 
 @Injectable()
 export class ProjectServiceService {
+    
+  _http;
 
-  constructor() { }
+  constructor(private http: Http) {
+    this._http = http;
+  }
+
+private handlError(error: Response){
+    let message = `Error status ${error.status} at ${error.url}`;
+    return Observable.throw(message);
+}
+
+getProjects(): Observable<any> {
+    return this._http.get('https://smart-city-lviv.herokuapp.com/api/projects')
+        .map((response: Response) => {
+            console.log('sadsafsa');
+            console.log(response.json());
+            return response.json();
+        }).catch(this.handlError);
+};
+
+getProject(id): Observable<any> {
+  return this._http.get('https://smart-city-lviv.herokuapp.com/api/projects/'+id)
+      .map((response: Response) => {
+          console.log('sadsafsa');
+          console.log(response.json());
+          return response.json();
+      }).catch(this.handlError);
+};
+
+postProject(project: any): Observable<any> {
+    const headers = new Headers({'Content-Type': 'application/json'});
```

Comment:
You can move this const to top level such as you're using it in a lot of methods over your class